src/unity/paths/operator_accounts_accountId_convert.yml

```diff
@@ -0,0 +1,36 @@
+patch:
+  operationId: ConvertAccountById
+  tags:
+    - Accounts
+  summary: Convert an account to contract
+  requestBody:
+    description: The contract start date
```

Comment:
Can/should this mention what the accepted format is? I'm assuming YYYY-MM-DD, but it would probably help the reader/consumer of the spec to know that for sure.